specification/marketplacecatalog/data-plane/Reviews/preview/2023-01-01-preview/reviews.json

```diff
@@ -0,0 +1,405 @@
+{
+  "x-generator": "NSwag v13.18.2.0 (NJsonSchema v10.8.0.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Rating And Reviews API",
+    "description": "Microsoft Marketplace Rating And Reviews API",
+    "version": "2023-01-01-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/products/{productId}/reviews": {
+      "get": {
+        "tags": [
+          "Reviews Public API"
+        ],
+        "summary": "Get reviews by unique product id",
+        "operationId": "Reviews_GetReviewsByOfferId",
```

Comment:
OperationId for get on a collection should contain "list"

Reply:
Please elaborate

Reply:
This is an Azure guideline to ensure consistent method names in client libraries generated from autorest. But I see that you are not using autorest (what are you using?) so this may not be important.

Reply:
Not using anything